api/docs/docs/access.mdx

```diff
@@ -3,95 +3,108 @@ id: access
 title: Accessing the API
 description: Register for the Covid Act Now API.
 ---
-import SignupForm from '@site/src/components/SignupForm';
 
-## Getting Started
+import SignupForm from "@site/src/components/SignupForm";
 
-<SignupForm />
+## Sign up
 
+<SignupForm />
 
-## Examples
+## Getting started
 
 In the examples below, replace `YOUR_KEY_HERE` with the API key from Step 1 above.
 
-### State summary data CSV
+The Covid Act Now API has coverage for states, counties, and metro areas (CBSAs). You can query the data in CSV or JSON formats.
 
-A CSV file containing the latest data and metrics for all states.
+Learn how to query:
 
-```
-https://api.covidactnow.org/v2/states.csv?apiKey=YOUR_KEY_HERE
-```
+- **[Current values for all states, counties, and metros](#current-values-for-all-states-counties-or-metros)**
+- **[Historic values for all states, counties, and metros](#historic-values-for-all-states-counties-or-metros)**
+- **[Current and historic values for an individual state, county, or metro area](#individual-locations)**
+
+### Current values for all states, counties, or metros
 
-[More details](/api#tag/State-Data/paths/~1states.csv?apiKey={apiKey}/get)
+These endpoints contain the latest available data for all states, counties, and cbsas (metros).
```

Comment:
To help readers understand that we mean a scalar, not the latest timeseries:

`.... latest available data (a scalar value per field) for all ...`